database/index.js

```diff
@@ -0,0 +1,104 @@
+var mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost:27017/fetcher');
+
+
+var productSchema = new mongoose.Schema({
+	uniqueID : Number, 
+	price : Number, 
+	quantity : Number, 
+	isPrime : String, 
+	inCart : Number, 
+	cartQuantity : Number
+})
+
+var Product = mongoose.model('Product', productSchema); 
+
+
+  var uniqueIDs = [ 1,   2,   3,   4,   5,   6,   7,   8,   9,   10,   11,   12,   13,
+  14,   15,   16,   17,   18,   19,   20,   21,   22,   23,   24,   25,
```

Comment:
This might be picking-nits, but arrays are improperly indented, and have too much whitespace